src/components/MdxComponents/Contributors.tsx

```diff
@@ -52,22 +86,56 @@ const Contributors = () => {
   `);
 
   const contributors = allContributor.nodes;
+
+  useEffect( ()=> {
+    // For accessing author by URL
+    if (searchParams.has('author')) {
+      const username = searchParams.get('author');
+      setSelectedAuthor(username);
+    }else {
+      setSelectedAuthor('');
+    }
+  },[]);
+
   return (
-    <div css={style(theme)}>
-      { contributors.map(contributor => {
-        return (
-          // eslint-disable-next-line react/jsx-no-target-blank
-          <a key={contributor.id} href={contributor.html_url} target='_blank'>
-            <img
-              alt={contributor.login}
-              src={contributor.avatar_url}
-            />
-            <span>{contributor.login}</span>
-          </a>
-        );
-      })
-      }
-    </div>
+    <>
+
+      <div css={style(theme)}>
```

Comment:
If the theme div is the outmost, you can remove `<> </>`.

Reply:
Thank you